### ISSUE TITLE
[Question] Uninstall simplewall

### ISSUE THREAD

**Issue by VirusRenamon** [henrypp/simplewall#584]
So I came across this issue on Windows 10. I had uninstalled this program from my computer seemingly fine until I downloaded new programs that needed internet access were mysteriously not connecting to the internet. Month later, a bunch of hair pulling out, and troubleshooting later, I found out its because all the programs that were not getting internet access was because of this program via system restore.

What I tried:
- No 3rd party Firewalls at all.
- Updating all my drivers (at least I updated my BIOS because of this incident)
- No AV at all.
- Safe Mode w/ Networking.
- Reinstalling programs.
- Making sure the host file is restored back to normal.
- Resetting windows firewall to default rules.
- Disabling windows firewall completely.
- Using a friends internet connection to make sure its not the router acting up.

Something about the uninstall process leaves something lingering to make new connections from newly installed programs impossible without this program.

**Comment by hwithyll** [comment]
Been over a month now. I have same issue. Any update?

**Comment by henrypp** [comment]
Do you "Disable filters" when uninstall SW?

ps: default uninstaller asking for this

**Comment by henrypp** [comment]
@hwithyll

**Comment by hwithyll** [comment]
Yes I did. Well, at least I certainly did the second time this happened. Let me try it again when i got time and report back. It was quite a while ago and I ended up having to re-install windows  What's the procedure if SW restrictions persist after uninstall? Is there a way to get previous set up running again? If I remember correctly, I tried just reinstalling SW last time and it failed to work.

**Comment by henrypp** [comment]
@hwithyll,

> What's the procedure if SW restrictions persist after uninstall?

Just "Disable filters", and thats all "procedure"!

> I tried just reinstalling SW last time and it failed to work.

Try find similar issues

**Comment by elimn** [comment]
I think I found the problem. I had this same issue. I repeatedly tried running `simplewall.exe /uninstall`, and when it prompted me I clicked "Disable Filters".  No luck.

I fixed this issue by opening the Simple Wall GUI, unchecking the "Protect Filters" boxes and "Run at Boot" in the menus, then clicked "Disable Filters" bnutton. My applications were no longer blocked. Then, I reran `simplewall.exe /uninstall` just to be save, and it didn't even prompt me to disable filters that time.

My guess is that the `/uninstall` command is no longer used/tested. Clicking `Disable Filters` in that doesn't work. Regardless, one of the the afore mentioned steps got SimpleWall fully disabled for me.
